deps/v8/src/platform-freebsd.cc

```diff
@@ -415,7 +415,7 @@ void* VirtualMemory::ReserveRegion(size_t size) {
   void* result = mmap(OS::GetRandomMmapAddr(),
                       size,
                       PROT_NONE,
-                      MAP_PRIVATE | MAP_ANON | MAP_NORESERVE,
+                      MAP_PRIVATE | MAP_ANON,
```

Comment:
For the record, this seems to be a backport of https://codereview.chromium.org/1025823003/, is that correct?